db/migrate/20220901071903_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] It would be better if we create the models for our migrations. Can you create a data model without the models ? :thinking: Kindly ask for a second opinion if I'm wrong

Reply:
@addod19 Yes it's possible. I created the models in the next project

Reply:
okay, cool :+1:

Reply:
@addod19 Thank you Sir